tests/rptest/services/redpanda.py

```diff
@@ -5493,6 +5493,22 @@ def estimate_bytes_written(self):
         else:
             return None
 
+    def estimate_bytes_read(self):
```

Comment:
nit: since this a common helper, a bit more descriptive name (like `estimate_disk_bytes_read` to distinguish from e.g. kafka bytes read) would be nice.